### ISSUE TITLE
Dogs self cleaning function broken

### ISSUE THREAD

**Issue by Buesi** [shafy/world#2]
**Describe the bug**
Dogs were designed as low maintenance companions for humans in the world. Unfortunately, the self cleaning function stopped working some 100 years ago and has never been fixed. Complaints of affected humans increased tremendously, they are reporting a bad odor and subsequent exhausting cleaning efforts.

**To Reproduce**
1. Take a dog, preferably from a shelter (adopt - don't shop)
2. Feed it, take it for a walk 3 times a day
3. Continue for 1-2 days without washing it
4. Smell the fur, if it is an especially malfunctioning one, you can even notice the problem when entering the room you have placed it in

**Expected behavior**
Self cleaning expected to happen 1-2/day. 

**Pictures**
No pictures available as feature has been broken for a long time.

**Additional context**
Maybe it helps to look into the source code of cats, the self cleaning feature seems to be working properly there.

**Comment by shafy** [comment]
Good catch. Unfortunately, we don't have the source code for cats (as you know). Might make sense to postpone this issue until we have recreated that, so we'll save us some work?

**Comment by Buesi** [comment]
Ok, I was at least able to document the desired function in a cat. Might help... 

<img width="897" alt="Screenshot 2020-07-19 at 16 44 19" src="https://user-images.githubusercontent.com/32681439/87877525-24c4f900-c9df-11ea-996f-3ea207580c24.png"
<img width="901" alt="Screenshot 2020-07-19 at 16 44 07" src="https://user-images.githubusercontent.com/32681439/87877528-28588000-c9df-11ea-86bd-e9ed65b20e9e.png">
